apps/opik-backend/src/main/java/com/comet/opik/infrastructure/redis/RedisUrlParser.java

```diff
@@ -0,0 +1,34 @@
+package com.comet.opik.infrastructure.redis;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public record RedisUrlParser(String scheme, String host, int port, int database) {
```

Comment:
Why do we need all these params? They are never used apart from tests. Only `database` is used.

Reply:
Because a dto with only the database index is not reusable